src/filters/blur.ts

```diff
@@ -36,6 +36,18 @@ export interface BlurOptions {
  */
 export function blur(image: Image, options: BlurOptions): Image {
   const { width, height } = options;
+
+  if (width < 1 || width % 2 === 0) {
+    throw new RangeError(
+      `Width must be an odd number and greater than 0. Got ${width}.`,
```

Comment:
"Got" is a bit informal and I checked other places and we use "Received" instead.

I prefer to have property names be referred to explicitly as named properties with quotes.

Example: `Invalid property "width". Must be an odd number greater than 0. Received 2.`